geopyspark/geotrellis/catalog.py

```diff
@@ -325,10 +329,10 @@ def query(layer_type,
                 All other types are restricted to ``Polygon`` and ``MulitPolygon``.
 
             If not specified, then the entire layer will be read.
-        time_intervals (list, optional): A list of strings that time intervals to query.
-            The strings must be in a valid date-time format. This parameter is only used when
-            querying spatial-temporal data. The default value is, None. If None, then only the
-            spatial area will be querried.
+        time_intervals (list, optional): A list of ``datetime.datetime`` instanaces that represent
```

Comment:
Honestly, I've never seen a convention around a `list` parameter. Though, I do think it'd be better if it was `[datetime.datetime]`.